Domains/3-DisciplineOther/Hydraulics/PipeNetworkHydraulicAnalysis/Released/PipeNetworkHydraulicAnalysis.01.00.00.ecschema.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- ==================================================================================
+|  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+|  * See LICENSE.md in the project root for license terms and full copyright notice. 
+======================================================================================= -->
+<ECSchema schemaName="PipeNetworkHydraulicAnalysis" alias="pipeNetAnlyt" version="01.00.00"
+	xmlns="http://www.bentley.com/schemas/Bentley.ECXML.3.2" displayLabel="Pipe Network Hydraulic Analysis" description="Pipe Network Hydraulic Analysis Schema.">
+	<ECSchemaReference name="CoreCustomAttributes" version="01.00.03" alias="CoreCA" />
+	<ECSchemaReference name="BisCustomAttributes" version="01.00.00" alias="bisCA"/>
+	<ECSchemaReference name="BisCore" version="01.00.10" alias="bis" />
+	<ECSchemaReference name="Analytical" version="01.00.01" alias="anlyt" />
+	<ECSchemaReference name="AecUnits" version="01.00.03" alias="AECU"/>
+	<ECSchemaReference name="RoadRailUnits" version="01.00.02" alias="rru"/>
+  <ECSchemaReference name="PipeworkPhysical" version="01.00.01" alias="pipphys"/>
+
+  <ECCustomAttributes>
+		<ProductionStatus xmlns="CoreCustomAttributes.01.00.03">
+			<SupportedUse>NotForProduction</SupportedUse>
```

Comment:
SupportedUse should be set to "FieldTesting" on the Released version

Reply:
done